subscriber/podaac_data_subscriber.py

```diff
@@ -334,18 +304,31 @@ def run():
     if args.verbose:
         print(str(results['hits'])+" new granules ingested for "+Short_Name+" since "+timestamp)
 
+    if args.dydoy or args.dymd:
+        try:
+            # file_start_times = [(splitext(r['meta']['native-id'])[0], datetime.strptime((r['umm']['TemporalExtent']['RangeDateTime']['BeginningDateTime'])), "%Y-%m-%dT%H:%M:%SZ") for r in results['items']]
+            file_start_times = [(r['meta']['native-id'], datetime.strptime((r['umm']['TemporalExtent']['RangeDateTime']['BeginningDateTime']), "%Y-%m-%dT%H:%M:%S.%fZ")) for r in results['items']]
+            print(file_start_times)
```

Comment:
debug statement? Not sure we need this.

Reply:
Just removed it, thanks for catching that.